packages/fxa-auth-server/lib/senders/renderer/bindings.ts

```diff
@@ -84,31 +91,28 @@ export abstract class LocalizerBindings {
     return { html, text, rootElement };
   }
 
-  protected async getComponent(type: string, name: string) {
+  protected async getComponent(type: ComponentType, name: string) {
     const path = `${this.opts.templates.basePath}/${type}/${name}`;
     const mjml = await this.fetchResource(`${path}/index.mjml`);
     const text = await this.fetchResource(`${path}/index.txt`);
     return { mjml, text };
   }
 
-  /**
-   * Returns the set of localization strings for the specified locale.
-   * @param locale Locale to use, defaults to en.
-   */
-  async fetchLocalizationMessages(locale?: string) {
-    // note: 'en' auth.ftl only exists for browser bindings / Storybook
-    // the fallback English strings within the templates will be shown in other envs
-    const path = `${this.opts.l10n.basePath}/${locale || 'en'}/auth.ftl`;
-
+  async getGlobalTemplateValues(template: string): Promise<Includes> {
     try {
-      return await this.fetchResource(path);
+      return import(`../emails/templates/${template}/includes.ts`);
```

Comment:
I'm open to suggestions on a better name here, or even a better approach. I personally much prefer all of the email copy living within each template directory.

I have this written in a comment in another spot, but it looks like we originally localized `subject` and `action` (if applicable) in the mailer method because we need a localized version of `subject` for the layout files and [the mailer](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/email.js#L508) (to be fair, I have no idea why we need this on emailConfig if it's in the layout), and sometimes we need a localized version of `action` to stick into `metadata.mjml`, inserted into a `script` tag. 

On top of this, to localize `mj-title` with the rest of the email would require a custom `mj-html-attributes` section which might be OK, but I have no idea how else we'd go about localizing what's going into the `script` tag.

Reply:
Ah, good point. I think my brain just went "TS files can use `import`".